client/application/application_manager.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2017 The Absolute Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default class ApplicationManager {
+
+  async isCromeBrowser(): Promise<boolean> {
+    var agent = navigator.userAgent.toLowerCase();
+    if(agent.indexOf('chrome') != -1){
```

Comment:
This is not perfect. For example, Samsung Internet, Opera, Whale browser will also include the `chrome` keyword in UA string.
Also, Android system WebView also includes `chrome` keyword in UA string.
You will have to investigate more things.

Reply:
@elliekwon, if you are in trouble with this, take a look at the previous commit. [link](https://github.com/lunchclass/absolute/blob/hackathon/client/deeplink/deeplink.js)